components/oscd-filter-button/src/oscd-filter-button.ts

```diff
@@ -0,0 +1,103 @@
+import {
+    css,
+    html,
+    TemplateResult,
+    query,
+    property,
+    unsafeCSS,
+  } from "lit-element";
+  
+  import "@material/mwc-icon-button";
+  import "@material/mwc-dialog";
+  
+  import "@openscd/oscd-filtered-list";
+  import { OscdFilteredList } from "@openscd/oscd-filtered-list";
+  import { Dialog } from "@material/mwc-dialog";
+  
+  export interface SelectedItemsChangedDetail {
+    selectedItems: string[];
+  }
+  export type SelectedItemsChangedEvent = CustomEvent<SelectedItemsChangedDetail>;
+  function newSelectedItemsChangedEvent(
+    selectedItems: string[],
+    eventInitDict?: CustomEventInit<SelectedItemsChangedDetail>
+  ): SelectedItemsChangedEvent {
+    return new CustomEvent<SelectedItemsChangedDetail>("selected-items-changed", {
+      bubbles: true,
+      composed: true,
+      ...eventInitDict,
+      detail: { selectedItems, ...eventInitDict?.detail },
+    });
+  }
+  
+  /**
+   * A mwc-list with mwc-textfield that filters the list items for given or separated terms
+   */
```

Comment:
missing JSDoc here and for every prop like how it's done in `oscd-components` 
https://github.com/openscd/oscd-components/blob/main/components/oscd-action-icon/src/oscd-action-icon.ts

Reply:
Out of scope